Gemfile

```diff
@@ -14,6 +14,7 @@ gem 'haml-rails', '~> 1.0'
 gem 'devise'
 gem 'omniauth-facebook', '~> 4.0'
 gem 'pundit'
+gem 'chewy'
```

Comment:
What does Chewy do?

Reply:
Chewy is an ODM and wrapper for the official Elasticsearch client and it provides functionality for Elasticsearch index handling, documents import mappings and chainable query DSL